src/lib/chains.ts

```diff
@@ -33,6 +33,99 @@ export const chainsNames = [
   'polygon',
 ] as const
 
+export const gasToken = {
```

Comment:
there's a `nativeCurrency` field in chains definitions, let's reuse it and update it if it's not correct